packages/babel-preset-env/test/normalize-options.spec.js

```diff
@@ -84,6 +84,24 @@ describe("normalize-options", () => {
         });
       });
     });
+
+    it("throws when including module plugins", () => {
+      expect(() =>
+        normalizeOptions.default({ include: ["proposal-dynamic-import"] }),
+      ).toThrow();
```

Comment:
Did this throw before this PR? If it didn't, I would prefer not to throw but to warn.

Reply:
Before the change it didn't throw, but index.js / filter-items.js could potentially have added a module plugin twice. For example the config that [I specified above](https://github.com/babel/babel/pull/10218#issue-297403129) did that. Users were also able to specify an ambiguous config with for example `modules` set to `cjs` and `includes` set to `['@babel/plugin-transform-modules-amd']`. Is it not better if preset-env throws in these cases in order to prevent wonky configs? I believe that there is otherwise a risk that users will overlook the warning. By throwing we are also preventing unnecessary support / issues further down the road for configs behaving weird due to inclusion of module plugins.